openapi-contracts/src/main/com/sinch/sdk/domains/verification/models/v1/webhooks/VerificationRequestEventResponseFlashCall.java

```diff
@@ -28,7 +28,7 @@ public interface VerificationRequestEventResponseFlashCall
 
   /**
    * The phone number that will be displayed to the user when the flash call is received on the
-   * user&#39;s phone. By default, the Sinch dashboard will randomly select the CLI that will be
+   * user's phone. By default, the Sinch dashboard will randomly select the CLI that will be
```

Comment:
I've seen some other &#39; in the PR #90 which are not replaced in this PR:
 -  openapi-contracts/src/main/com/sinch/sdk/domains/verification/models/v1/VerificationMethod.java
 - openapi-contracts/src/main/com/sinch/sdk/domains/verification/models/v1/VerificationStatus.java

Reply:
Right.
This is coming from generator and dedicated redocly's enum descriptions injection
Working on it; thank you to have highlighted this